bin/dotd

```diff
@@ -109,118 +163,122 @@ def content_push(server_name, environment_name, user_name)
   end
 end
 
-def open_pull_request(base, compare, title)
-  open_url "https://github.com/code-dot-org/code-dot-org/compare/#{base}...#{compare}?expand=1&title=#{CGI.escape title}"
-end
-
 # Prints the commit messages of commits present in branch head but not branch
-# base.
+# base. Truncates all such messages to the first line.
 # @param base [String] The base branch to compare.
 # @param head [String] The head branch to compare against.
 # @example For a DTT: view_commits(base: 'test', head: 'staging')
 def view_commits(base:, head:)
+  puts "  Commits in #{head} to be merged into #{base}"
+  puts ''
   GitHub.compare(base: base, head: head).each do |commit_message|
-    # Empirically, commit messages often contain blank lines. For console
-    # readability, we attempt to remove them.
-    puts commit_message.gsub("\n\n", "\n")
+    # Truncate the message to its first line.
+    commit_message_truncated = "    #{commit_message[/.*/].chomp}"
+    puts dim(commit_message_truncated).to_s
   end
+  puts ''
 end
 
-puts <<-EOS.unindent
-
-  #{bold 'Welcome to Robo-Dev-of-the-Day!'}
-
-    d[ o_0 ]b
-
-  #{bold 'Documentation'}
-
-    Dev-of-the-Day Checklist
-    #{dim 'http://wiki.code.org/display/PROD/Dev+of+the+Day+Checklist'}
+# Prints the DTT subtopic, formatted based on its contents.
+def print_dtt_subtopic
+  if DevelopersTopic.dtt?
+    output_text = dim "CURRENT DTT SUBTOPIC: #{DevelopersTopic.dtt_subtopic}"
+  else
+    output_text = red "CURRENT DTT SUBTOPIC: #{DevelopersTopic.dtt_subtopic}"
+  end
+  puts output_text
```

Comment:
A few recommendations:
1. I would flip this around (see example below)
2. "Subtopic" isn't a super well-known term on the team, maybe use something a little simpler.
3. I'm not sure the screaming caps are needed, esp. with the red variant when DTT is no.

Proposal:
```ruby
def print_dtt_subtopic
  msg = "The #developers room says DTT: #{DevelopersTopic.dtt_subtopic}"
  if DevelopersTopic.dtt?
    puts dim msg
  else
    puts red msg
  end
end
```